src/App.tsx

```diff
@@ -1,23 +1,23 @@
-import React from 'react';
-import UserList from './components/UserList'
-import NameForm from './components/NameForm'
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
-import { User } from './types';
+import Home from "routes/Home";
+import Users from "routes/Users/Index";
```

Comment:
if you name the file `index` (lowercase), you can import it like 
```suggestion
import Users from "routes/Users";
```